.github/workflows/_ci.yaml

```diff
@@ -612,34 +619,205 @@ jobs:
   #       pytest-report.jsonl
   #   secrets: inherit
 
-  test-gemma:
-    needs: build-gemma
-    uses: ./.github/workflows/_test_unit.yaml  
+  # test-gemma:
+  #   needs: build-gemma
+  #   uses: ./.github/workflows/_test_unit.yaml  
+  #   if: inputs.ARCHITECTURE == 'amd64'
+  #   with:
+  #     TEST_NAME: gemma
+  #     EXECUTE: |
+  #       docker run --shm-size=1g --gpus all ${{ needs.build-gemma.outputs.DOCKER_TAG_FINAL }} \
+  #       bash -ec \
+  #       "cd /opt/gemma && pip install -e .[dev] && pytest ." | tee test-gemma.log
+  #     STATISTICS_SCRIPT: |
+  #       summary_line=$(tail -n1 test-gemma.log)
+  #       errors=$(echo $summary_line | grep -oE '[0-9]+ error' | awk '{print $1} END { if (!NR) print 0}')
+  #       failed_tests=$(echo $summary_line | grep -oE '[0-9]+ failed' | awk '{print $1} END { if (!NR) print 0}')
+  #       passed_tests=$(echo $summary_line | grep -oE '[0-9]+ passed' | awk '{print $1} END { if (!NR) print 0}')
+  #       total_tests=$((failed_tests + passed_tests))
+  #       echo "TOTAL_TESTS=${total_tests}" >> $GITHUB_OUTPUT
+  #       echo "ERRORS=${errors}" >> $GITHUB_OUTPUT
+  #       echo "PASSED_TESTS=${passed_tests}" >> $GITHUB_OUTPUT
+  #       echo "FAILED_TESTS=${failed_tests}" >> $GITHUB_OUTPUT
+  #     ARTIFACTS: |
+  #       test-gemma.log
+  #   secrets: inherit
+
+  # test-maxtext:
+  #   needs: build-maxtext
+  #   if: inputs.ARCHITECTURE == 'amd64' # no arm64 gpu runners
+  #   uses: ./.github/workflows/_test_maxtext.yaml
+  #   with:
+  #     MAXTEXT_IMAGE: ${{ needs.build-maxtext.outputs.DOCKER_TAG_FINAL }}
+  #   secrets: inherit
+
+  test-axlearn-eks:
+    needs: build-axlearn
     if: inputs.ARCHITECTURE == 'amd64'
-    with:
-      TEST_NAME: gemma
-      EXECUTE: |
-        docker run --shm-size=1g --gpus all ${{ needs.build-gemma.outputs.DOCKER_TAG_FINAL }} \
-        bash -ec \
-        "cd /opt/gemma && pip install -e .[dev] && pytest ." | tee test-gemma.log
-      STATISTICS_SCRIPT: |
-        summary_line=$(tail -n1 test-gemma.log)
-        errors=$(echo $summary_line | grep -oE '[0-9]+ error' | awk '{print $1} END { if (!NR) print 0}')
-        failed_tests=$(echo $summary_line | grep -oE '[0-9]+ failed' | awk '{print $1} END { if (!NR) print 0}')
-        passed_tests=$(echo $summary_line | grep -oE '[0-9]+ passed' | awk '{print $1} END { if (!NR) print 0}')
+    runs-on: eks
+    env:
+      AXLEARN_DOCKER_IMAGE: ${{ needs.build-axlearn.outputs.DOCKER_TAG_FINAL }}
+      JOB_NAME: axlearn-${{ github.run_id }}
+      TOKEN_NAME: axlearn-${{ github.run_id }}-token
+    steps:
+    - name: Set date env var for saving files
+      run: |
+        echo "DATE_TEST_RAN=$(date +'%Y-%m-%d-%H-%M-%S')" >> $GITHUB_ENV
```

Comment:
Suggest using the run_id or similar rather than inventing something new and race-prone.